client.go

```diff
@@ -155,6 +155,18 @@ func (client *Client) DeleteSerie(key string) (err error) {
 	return err
 }
 
+// DeleteRange - Delete data points for a given timeseries and interval range in the form of start and end delete timestamps.
+// args:
+//  key - time series key name
+//  fromTimestamp - start of range. You can use TimeRangeMinimum to express the minimum possible timestamp.
+//  toTimestamp - end of range. You can use TimeRangeFull or TimeRangeMaximum to express the maximum possible timestamp.
+func (client *Client) DeleteRange(key string, fromTimestamp int64, toTimestamp int64) (err error) {
```

Comment:
@filipecosta90 you should adapt it to the new return value (i64)

Reply:
@gkorland adjusted towards the new reply type.